docs/topics/proc_running-the-containerless-mta-cli.adoc

```diff
@@ -26,26 +26,46 @@ In {ProductShortName} 7.2.0 and later, containerless CLI is a default mode. To e
 * You have the `JAVA_HOME` environmental variable set.
 * You have the `JVM_MAX_MEM` system variable set.
 +
-NOTE: If you do not set `JVM_MAX_MEM`, the analysis might hang.
+[NOTE]
+====
+If you do not set `JVM_MAX_MEM`, the analysis might hang.
+====
 
-.Procedure
+.Procedure for Linux
 
-. Move the installed {ProductShortName} CLI `.zip` file to your `$PATH`:
+After unpacking the whole zip file to `~/.kantra`, either add that path to the `$PATH` variable or move the `mta-cli` binary to `/usr/bin`.
```

Comment:
I think we should only mention moving the binary to the PATH, and remove the part about moving it to `/usr/bin`. That path was only used as an example, and does not hold any significance.